Playbooks/Enrich-CIRCL-hashlookup/Playbook/azuredeploy.json

```diff
@@ -0,0 +1,514 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "resourceTags": {
+            "type": "object",
+            "defaultValue": {
+                "LogicAppsCategory": "security"
+            }
+        },
+        "PlaybookName": {
+            "defaultValue": "Enrich-CIRCL-hashlookup",
+            "type": "string"
+        },
+        "workflows_Enrich_CIRCL_hashlookup_name": {
+            "defaultValue": "Enrich-CIRCL-hashlookup",
+            "type": "String"
+        },
```

Comment:
Parameter "workflows_Enrich_CIRCL_hashlookup_name" looks redundant. PlaybookName can be used at its place as it is similar to PlaybookName parameter.

Reply:
fixed